test/boulder-config.json

```diff
@@ -31,21 +31,58 @@
   },
 
   "wfe": {
-    "listenAddress": "0.0.0.0:4000"
+    "listenAddress": "127.0.0.1:4000"
```

Comment:
You don't have to do this one for me; I just wanted the integration test (in particular) to not be open to the world.